include/LightGBM/c_api.h

```diff
@@ -324,293 +343,333 @@ LIGHTGBM_C_EXPORT int LGBM_DatasetSetField(DatasetHandle handle,
                                            int type);
 
 /*!
-* \brief get info vector from dataset
-* \param handle a instance of data matrix
-* \param field_name field name
-* \param out_len used to set result length
-* \param out_ptr pointer to the result
-* \param out_type  C_API_DTYPE_FLOAT32 or C_API_DTYPE_INT32
-* \return 0 when succeed, -1 when failure happens
+ * @fn LGBM_DatasetGetField
+ * @brief Get info vector from dataset.
+ * @param handle An instance of data matrix
+ * @param field_name Field name
+ * @param out_len Used to set result length
+ * @param out_ptr Pointer to the result
+ * @param out_type C_API_DTYPE_FLOAT32 or C_API_DTYPE_INT32
+ * @return 0 when succeed, -1 when failure happens
 */
 LIGHTGBM_C_EXPORT int LGBM_DatasetGetField(DatasetHandle handle,
                                            const char* field_name,
                                            int* out_len,
                                            const void** out_ptr,
                                            int* out_type);
 
-
 /*!
-* \brief Update parameters for a Dataset
-* \param handle a instance of data matrix
-* \param parameters parameters
+ * @fn LGBM_DatasetUpdateParam
+ * @brief Update parameters for a Dataset.
+ * @param handle An instance of data matrix
+ * @param parameters Parameters
 */
 LIGHTGBM_C_EXPORT int LGBM_DatasetUpdateParam(DatasetHandle handle, const char* parameters);
 
 /*!
-* \brief get number of data.
-* \param handle the handle to the dataset
-* \param out The address to hold number of data
-* \return 0 when succeed, -1 when failure happens
+ * @fn LGBM_DatasetGetNumData
+ * @brief Get number of data.
+ * @param handle The handle to the dataset
+ * @param[out] out The address to hold number of data
+ * @return 0 when succeed, -1 when failure happens
 */
 LIGHTGBM_C_EXPORT int LGBM_DatasetGetNumData(DatasetHandle handle,
                                              int* out);
 
 /*!
-* \brief get number of features
-* \param handle the handle to the dataset
-* \param out The output of number of features
-* \return 0 when succeed, -1 when failure happens
+ * @fn LGBM_DatasetGetNumFeature
+ * @brief Get number of features.
+ * @param handle The handle to the dataset
+ * @param out The output of number of features
```

Comment:
` * @param[out] out The address to hold  number of features`
For the consistency with the function above.